model.py

```diff
@@ -1,23 +1,24 @@
 from dao import Dao
-
-collName='students'
-
+import sys
 class Model:
-
-    #def save
-    #def get_by_params
   
     def __init__(self, collection):
-        self.id = collection['id'] if 'id' in collection else None
+        self._id = str(collection['_id'])
+        self.name = collection['name']
+        
 
     @staticmethod
     def get_by_id(student_Id):
         dao = Dao()
         try:
```

Comment:
no need for try / except in model, let the APP handle all error handling .. that is why you are catching multiple exceptions